src/Arcus.Messaging.Pumps.ServiceBus/AzureServiceBusMessagePump.cs

```diff
@@ -163,7 +192,7 @@ protected virtual async Task CompleteMessageAsync(string lockToken)
 
             if (_messageReceiver == null)
             {
-                throw new InvalidOperationException("Message receiver is not initialized yet.");
+                throw new InvalidOperationException($"[Job: {JobId}] Message receiver is not initialized yet");
```

Comment:
We shouldn't add these prefixes to our exceptions

Reply:
👍